zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -125,7 +146,10 @@ def get_start_request(self, url):
         meta = {
             "crawling_logs": {"page_type": "productNavigation"},
         }
-        if self.args.crawl_strategy == EcommerceCrawlStrategy.full:
+        if self.args.crawl_strategy == EcommerceCrawlStrategy.full or (
+            self.args.crawl_strategy == EcommerceCrawlStrategy.automatic
+            and is_homepage(url)
```

Comment:
What do you think about adding a log message, so that from the logs it's clear if a spider thinks a seed URL is a home page or not?

Reply:
Sounds good! https://github.com/zytedata/zyte-spider-templates/pull/52/commits/bffc9593daa5700294e99072c330b02225c9f7b8

Reply:
* I think it's best to have logging in the spider, not in the utility function. 
* Not sure about this: should it be DEBUG or INFO? How are similar messages logged?

Reply:
What do you think about this? https://github.com/zytedata/zyte-spider-templates/pull/52/commits/b4ffc303e6380e307ab8fb253e8463401ff8a9ef

I think adding a bit of explanation to the automatic strategy would help in the long run, especially when we're considering other ways to add more heuristics like sitemaps later on.

We can use INFO so the logs would be available by default to Scrapy Cloud users. It's also similar to how the other logs in the spider where it uses INFO to indicate/clarify some spider behaviors.